src/content/guides/code-splitting.md

```diff
@@ -76,7 +77,8 @@ console.log(
 );
 ```
 
-__webpack.config.js__
+__webpack.config.jsFor more detailed setup please install refer this guide https://www.robinwieruch.de/minimal-react-webpack-babel-setup/
```

Comment:
```suggestion
__webpack.config.js__
```

Reply:
`For more detailed setup please install refer this guide https://www.robinwieruch.de/minimal-react-webpack-babel-setup/`
Ig moving it [here](https://github.com/webpack/webpack.js.org/blob/master/src/content/guides/code-splitting.md#prefetchingpreloading-modules) would be better.

Reply:
You forgot to re-insert the respective content as above.